rasa/core/training/dsl.py

```diff
@@ -197,6 +197,39 @@ def __init__(self, domain, interpreter, template_vars=None, use_e2e=False):
 
         return story_steps
 
+    def verify_stories_format(self, filename):
+        story_lines = self._remove_comments(filename)
+
+        for line in range(len(story_lines)):
+            story_lines[line] = story_lines[line].strip()
```

Comment:
why are we stripping? Also you can put this into the second loop so we don't have to loop twice.